prime-router/docs/proposals/0017-serializer-error-codes.md

```diff
@@ -240,4 +234,5 @@ Version 2 (My preferred table)
 | GENERAL FIELD              | INVALID_HL7_GENERAL_PARSE                | field was unable to be parsed by HAPI or ReportStream        |
 | GENERAL MSG                | INVALID_HL7_MSG_TYPE_MISSING             | Missing required HL7 message type field                      |
 | GENERAL MSG                | INVALID_HL7_MSG_TYPE_UNSUPPORTED         | Unsupported HL7 message type                                 |
-| GENERAL MSG                | INVALID_HL7_MSG_FORMAT_INVALID           | Invalid HL7 message format                                   |
\ No newline at end of file
+| GENERAL MSG                | INVALID_HL7_MSG_FORMAT_INVALID           | Invalid HL7 message format                                   |
+| MISC                       | UNKNOWN                                  | Error cannot be determined                                   |
```

Comment:
Dumb question, but is there anything else useful we can send in addition to the other ConsolidatedActionLog fields in the hopefully rare case we can't determine the error?  I'm thinking not since that's ultimately the problem, but I'm just wondering if there's anything else we could help end users diagnose issues on their end or help us identify them if they were to reach out to us.

I guess as long as we have a request or message id that should be enough for us to look through the logs for any errors 🤔 🤔 🤔

Reply:
Hey, I got Unknown right here! 

This error code is mostly so we can set it as the default value for FieldProcessingMessage's errorCode field, I figure its better than setting it as "null". I don't actually see how we would end up with this error code if the pattern proposed in the design is followed.

Reply:
Gotcha, definitely makes sense.  Thanks for following up!